syft/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.19a1"

```

Comment:
Could you add this location to setup.cfg as well for bump version support

Reply:
Done.